Source/SantaGUI/SNTMessageWindowController.m

```diff
@@ -109,24 +159,24 @@ - (void)loadWindow {
 
 - (IBAction)showWindow:(id)sender {
   [(SNTMessageWindow *)self.window fadeIn:sender];
+  [self addObserver:self forKeyPath:@"self.event.fileBundleHash" options:0 context:NULL];
 }
 
-- (IBAction)closeWindow:(id)sender {
+- (void)closeWindow:(id)sender {
```

Comment:
closeWindow is being using throughout the entire SNTMessageWindowcontroller. When I change it to what you suggestion and trigger a SantaBlock, I am unable to click the buttons